python_examples/intervention_example/social_distancing_simulation_flow.py

```diff
@@ -0,0 +1,129 @@
+#
+# Example simulation script with social distancing intervention data output
+# and visualisation
+#
+
+import os
+import logging
+import pandas as pd
+import matplotlib.pyplot as plt
+
+import pyEpiabm as pe
+
+# Setup output for logging file
+logging.basicConfig(filename='sim.log', filemode='w+', level=logging.DEBUG,
+                    format=('%(asctime)s - %(name)s'
+                            + '- %(levelname)s - %(message)s'))
+
+# Set population input file
+file_loc = os.path.join(os.path.dirname(__file__), "input.csv")
+
+# sim_params give details for the running of the simulations
+sim_params = {"simulation_start_time": 0, "simulation_end_time": 50,
+              "initial_infected_number": 1, "initial_infect_cell": True}
+
+# Set parameter file
+name_parameter_file = 'social_distancing_parameters.json'
+
+# Set config file for Parameters
+pe.Parameters.set_file(os.path.join(os.path.dirname(__file__),
+                       name_parameter_file))
+
+# Parameter to change
+to_modify_parameter_values = {'distancing_enhanced_prob': [1, 0.5, 0],
+                              'distancing_spatial_susc': [0.1, 0.3, 0.9]}
+for to_modify_parameter, parameter_values in to_modify_parameter_values.\
+        items():
+    for parameter_value in parameter_values:
+        name_output_file = 'output_{}_{}.csv'.format(
+                parameter_value, to_modify_parameter)
+
+        if to_modify_parameter == 'distancing_enhanced_prob':
+            pe.Parameters.instance().intervention_params['social_distancing'][
+                to_modify_parameter] = [parameter_value] * 17
```

Comment:
What is this 17?

Reply:
Changed numerical value 17 to num_age_group :)